pkg/resources/k8sgpt.go

```diff
@@ -194,6 +194,14 @@ func GetDeployment(config v1alpha1.K8sGPT) (*appsv1.Deployment, error) {
 									Name:  "K8SGPT_BACKEND",
 									Value: string(config.Spec.Backend),
 								},
+								{
+									Name:  "XDG_CONFIG_HOME",
+									Value: "/k8sgpt-config/",
+								},
+								{
+									Name:  "XDG_CACHE_HOME",
```

Comment:
I think it makes sense to have a different path for cache, e.g `/k8sgpt-config/.cache` and use subPath in the ConfigMap.

Reply:
@arbreezy what do you think about my last namings? I use k8sgpt-data for the volume mountpath and .config and .cache sub directories

Reply:
LGTM !